pyrk/neutronics.py

```diff
@@ -84,15 +111,25 @@ def dpdt(self, t_idx, components, power, zetas):
         :type power: float.
         :param zetas: the current delayed neutron precursor populations, zeta_i
         :type zetas: np.ndarray.
+        :param rho_r: sum of the reactivity gain by all the reflectors
+        :type rho_r: float
         """
         rho = self.reactivity(t_idx, components)
         beta = self._pd.beta()
         lams = self._pd.lambdas()
-        Lambda = self._pd.Lambda()
+        Lambda = self._Lambda
```

Comment:
It is interesting to choose to decouple lambda from the delayed neutron precursor data. I think this is ok, except there are now many Lambdas to keep track of, Lambda and lambda_ref... not to mention ref_lambda...